### PR TITLE
fix: Icon pencil closer to title

### LINKED ISSUE #918: Pencil icon to edit cell name is really far away from the text
user commented that the pencil icon is really far from the text of the cell title. Is there a way we can bring those closer together?

image

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #918 

Before
Screen Shot 2021-04-22 at 1 42 08 PM

After

Screen Shot 2021-04-22 at 1 40 59 PM

Both are the same for ipad sized devices

Screen Shot 2021-04-22 at 1 43 48 PM
Screen Shot 2021-04-22 at 1 44 24 PM

For squished computer screen
Screen Shot 2021-04-22 at 1 44 40 PM

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 62 | :yellow_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?vars%5Blighthouse_score_branch%5D=1276)

**Review by Asalem1 — COMMENTED** [review]
Is there a picture / gif that can be used as a reference to see how this looks on different sized screens?

File: src/pageLayout/components/RenamablePageTitle.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@ $renamable-page-title--padding: 0;
 
 .renamable-page-title {
   width: 100%;
+  max-width: 850px;
```
Comment (hoorayimhelping) [inline src/pageLayout/components/RenamablePageTitle.scss:11]: if you remove the max width and instead add:

```css
display: inline-block;
```

does that fix the issue?

Comment (zoesteinkamp) [reply src/pageLayout/components/RenamablePageTitle.scss:11]: let me see what that does

Comment (zoesteinkamp) [reply src/pageLayout/components/RenamablePageTitle.scss:11]: no it results in it going back to the old style

**Comment by zoesteinkamp** [comment]
> Is there a picture / gif that can be used as a reference to see how this looks on different sized screens?

**Comment by zoesteinkamp** [comment]
Pictures now added

**Review by russorat — APPROVED** [review]
LGTM

**Review by hoorayimhelping — APPROVED** [review]

**Review by juliajames12 — APPROVED** [review]
Looks great!

**Pull request merged**